src/convertResults.ts

```diff
@@ -11,12 +11,6 @@ export function convertResults(report: CLIEngine.LintReport): ToolResult[] {
     const messages = result.messages
     const fatalErrors = messages
       .filter((m) => m.fatal)
-      // this filter is here to avoid errors related to files not supported by current parser
-      // example: Using a "@typescript-eslint/parser" parser does not allow JSON files to be analyzed
-      .filter(
-        (m) =>
-          !m.message.includes("The file does not match your project config:")
-      )
```

Comment:
Why did you remove this part?

Reply:
I think we don't need it anymore now that we filter out typescript patterns for other file types.
https://github.com/codacy/codacy-eslint/commit/d182d03278f62001118d46a61a192488e78e3557#diff-8ebafde9572c134640c7265425ec4eefR61
Do you think we should keep it anyways?

Reply:
So you're saying that we are maintain the workaround rate constant, again.. ?
Cool!